main.go

```diff
@@ -282,7 +287,10 @@ func main() {
 	etcdCtrl := NewETCDController(agents, etcd, loadbalancers, lbUpdates, metrics, etcdCycleCh, nil)
 
 	dnsCycleCh := make(chan time.Time, 0)
-	dnsCtrl := NewDNSController(dnsProvider, lbUpdates, metrics, dnsCycleCh, nil)
+	var DNSControllerSync = time.Duration(syncIntervalDNSController) * time.Second
+	var DNSControllerDownloadSync = time.Duration(syncIntervalDNSController) * time.Second
```

Comment:
un-exported names should be lowercase

Reply:
I think all the issues are now resolved.